app/main.py

```diff
@@ -1,8 +1,43 @@
+from typing import List
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark < self.clean_power:
+            cost = (
+                car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating
+            ) / self.distance_from_city_center
```

Comment:
There is a potential division by zero error here if `distance_from_city_center` is zero. Consider adding a check to ensure this value is not zero before performing the division.